app/assets/javascripts/select-kit/components/topic-notifications-button.js.es6

```diff
@@ -28,6 +28,18 @@ export default Component.extend({
   },
 
   _changeTopicNotificationLevel(level) {
+    const topicSectionNode = level.event.target.querySelector("#topic");
+    if (topicSectionNode && topicSectionNode.getAttribute("data-topic-id")) {
```

Comment:
I think it's cleaner to use dataset to access data attributes like `topicSectionNode.dataset.topicId`